contrib/Dockerfile.example

```diff
@@ -0,0 +1,16 @@
+FROM cloudposse/geodesic:0.2.0
```

Comment:
Annotated and placed `Dockerfile` comments on new lines since `Dockerfile` does not support inline comments